### PR TITLE
Silently fail if .NET CLI isn't installed

### PR DESCRIPTION
**Pull request by gregg-miskelly**
This checkin adds code to the debugger downloader to detect if the .NET CLI is installed. If not, it will silently exit without trying to download the debugger.

### CODE REVIEW

**Comment by gregg-miskelly** [comment]
@DustinCampbell and @chuckries please review.

File: src/coreclr-debug.ts [hunk 1]
```diff
@@ -1,194 +1,228 @@
...
+
+// Determines if the specified command is in one of the directories in the PATH environment variable.
+function isOnPath(command : string) : boolean {
+    let pathValue = process.env['PATH'];
+    if (!pathValue) {
+        return false;
+    }
+    let fileName = command;
+    let seperatorChar = ':';
+    if (process.platform == 'win32') {
```
Comment (DustinCampbell) [inline src/coreclr-debug.ts]: ===

**Comment by DustinCampbell** [comment]
I'm not sure why the diff shows this file as being removed and re-added, but :+1:

**Comment by chuckries** [comment]
:+1:  looks good. did the default encoding or tab/space behavior change somehow? I am also curious why the entire file shows as diff'd.

**Comment by DustinCampbell** [comment]
OK. With @chuckries's sign off, I'll go ahead and merge.

**Pull request merged**